Makefile

```diff
@@ -25,9 +25,9 @@ install:
 	go install
 
 get-index:
-	docker login registry.redhat.io && \
-	docker rmi $(IMAGE) | true
-	docker run --rm --entrypoint cat $(IMAGE) /database/index.db > \
+	$(CONTAINER_ENGINE) login registry.redhat.io && \
+	$(CONTAINER_ENGINE) rmi -f $(IMAGE) | true
```

Comment:
Perhaps we should do a check here on what's available online before we wipe the image and force the largish download - possibly replacing it with nothing new?

Reply:
skopio can do this kind of check without a download I think, but I should say I'm not sure how to check with generic container-engine...